kproject.lock

```diff
@@ -0,0 +1,5 @@
+https://github.com/Quillraven/Fleks.git/src#c24925091ced418bf045ba0672734addaab573d8 ::: c24925091ced418bf045ba0672734addaab573d8:fe020a0bf50ff6f3fbf706f10af80b05fbf2050005010d04f40e0b0b02020ef7
+https://github.com/korlibs/korge-fleks.git/fleks/kproject.yml#02fed30e752cc14c71cecafbdee2882c95d99e64 ::: 02fed30e752cc14c71cecafbdee2882c95d99e64:070307f0ff0802f9f506fd0ff6f4f9f40807f8fbfdfc0af4fc0bf9090e05f302
+https://github.com/korlibs/korge-fleks.git/korge-fleks#02fed30e752cc14c71cecafbdee2882c95d99e64 ::: 02fed30e752cc14c71cecafbdee2882c95d99e64:0c070dfcf90005f2f9060d060df2f7f8ff0ef3f905f7f8fef5f9fbfc0d0d0cf4
+https://github.com/korlibs/korge-parallax.git/korge-parallax#994e0356761e6f19cb518e39332ac9383bd8149f ::: 994e0356761e6f19cb518e39332ac9383bd8149f:01f10a0203f0f607fcfdfaf8fe0c0ffe0f09fdf3f3f1f2080803f30b0b00fcf8
+https://github.com/korlibs/korge-tiled.git/korge-tiled#734d96ccc18733064ef9fbda8ac359585011112d ::: 734d96ccc18733064ef9fbda8ac359585011112d:0701f1f7fa0ffbf3f60506f4f90c06fb010b0e03fdf70c0b0afd02f5f8fbfbff
```

Comment:
@soywiz  For what do we need this file? And should it be checked in or put in .gitignore and used only locally?

Reply:
This file is automatically generated. This is the equivalent of yarn.lock.
Lets suppose a malicious person takes the control of a git repository we are including and changes sn existing tag to a commit with malicious content. This lock file would detect that change and prevent the code from loading.

Reply:
This is a real threat. Specially when people starts sharing their code:
* https://blog.jetbrains.com/kotlin/2021/10/important-ua-parser-js-exploit-and-kotlin-js/
* https://blog.jetbrains.com/kotlin/2021/10/control-over-npm-dependencies-in-kotlin-js/

Reply:
Ah, thanks for the explanation. Good points! Now it's clear :)